src/vos/vos_aggregate.c

```diff
@@ -1198,6 +1206,11 @@ fill_segments(daos_handle_t ih, struct agg_merge_window *mw,
 	unsigned int		 i, scm_max;
 	int			 rc = 0;
 
+	if (io->ic_seg_cnt == 0) {
+		/** No logical extents to handle (only removals) */
```

Comment:
Not only 'no logical extents', but also 'no truncated physical extents' to handle.

Reply:
True